### PR TITLE
[Doc]add more description in kafka connector and modify the name for op field

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

**Review by EsoragotoSpirit — DISMISSED** [review]

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 1]
```diff
@@ -65,6 +65,12 @@ CREATE TABLE test_tbl (id INT, city STRING);
 
 1. Configure the Kafka connector. In the **config** directory under the Kafka installation directory, create the configuration file **connect-StarRocks-sink.properties** for the Kafka connector, and configure the following parameters. For more parameters and descriptions, see [Parameters](#Parameters).
 
+    :::note
+
+    The Kafka connector is a sink connector。
```
Comment (evelynzhaojie) [inline docs/en/loading/Kafka-connector-starrocks.md:70]: Chinese period

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 2]
```diff
@@ -321,5 +341,5 @@ transforms.unwrap.delete.handling.mode
 
 In the above configurations, we specify `transforms=addfield,unwrap`.
 
-- The addfield transform is used to add the __op field to each record of Debezium CDC-formatted data to support the StarRocks Primary Key table. If the StarRocks table is not a Primary Key table, you do not need to specify the addfield transform. The addfield transform class is com.Starrocks.Kafka.Transforms.AddOpFieldForDebeziumRecord. It is included in the Kafka connector JAR file, so you do not need to manually install it.
+- If the StarRocks table is a Primary Key table, you need to specify the addfield transform to add an op field to each record of the Debezium CDC formatted data. If the StarRocks table is not a Primary Key table, you do not need to specify the addfield transform. The addfield transform class is com.Starrocks.Kafka.Transforms.AddOpFieldForDebeziumRecord. It is included in the Kafka connector JAR file, so you do not need to manually install it.
```
Comment (evelynzhaojie) [inline docs/en/loading/Kafka-connector-starrocks.md:344]: use the codeph format for `addfield` and `op` would be better

File: docs/zh/loading/Kafka-connector-starrocks.md [hunk 3]
```diff
@@ -221,5 +237,5 @@ transforms.unwrap.delete.handling.mode=rewrite
 
 在上述配置中，我们指定 `transforms=addfield,unwrap`。
 
-- addfield transform 用于向 Debezium CDC 格式数据的每个记录添加一个__op字段，以支持 [主键表](../table_design/table_types/primary_key_table.md)，。如果 StarRocks 表不是主键表，则无需指定 addfield 转换。addfield transform 的类是 com.Starrocks.Kafka.Transforms.AddOpFieldForDebeziumRecord，已经包含在 Kafka connector 的 JAR 文件中，您无需手动安装。
+- 如果 StarRocks 表是主键表，则需要指定 addfield transform，用于向 Debezium CDC 格式数据的每个记录添加一个 `op` 字段。如果 StarRocks 表不是主键表，则无需指定 addfield transform。addfield transform 的类是 com.Starrocks.Kafka.Transforms.AddOpFieldForDebeziumRecord，已经包含在 Kafka connector 的 JAR 文件中，您无需手动安装。
```
Comment (amber-create) [inline docs/zh/loading/Kafka-connector-starrocks.md:240]: image
加 code 格式，英文同理

**Review by amber-create — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

File: docs/zh/loading/Kafka-connector-starrocks.md [hunk 4]
```diff
@@ -65,6 +65,11 @@ CREATE TABLE test_tbl (id INT, city STRING);
 
 1. 配置 Kafka connector。在 Kafka 安装目录下的 **config** 目录，创建 Kafka connector 的配置文件 **connect-StarRocks-sink.properties**，并配置对应参数。参数和相关说明，参见[参数说明](#参数说明)。
 
+  :::note
+
+  Kafka connector 是 sink connector。
```
Comment (yingtingdong) [inline docs/zh/loading/Kafka-connector-starrocks.md:70]: StarRocks kafka connector，需要说明吗？我们只提供了 sink connector。

File: docs/zh/loading/Kafka-connector-starrocks.md [hunk 5]
```diff
@@ -221,5 +237,5 @@ transforms.unwrap.delete.handling.mode=rewrite
 
 在上述配置中，我们指定 `transforms=addfield,unwrap`。
 
-- addfield transform 用于向 Debezium CDC 格式数据的每个记录添加一个__op字段，以支持 [主键表](../table_design/table_types/primary_key_table.md)，。如果 StarRocks 表不是主键表，则无需指定 addfield 转换。addfield transform 的类是 com.Starrocks.Kafka.Transforms.AddOpFieldForDebeziumRecord，已经包含在 Kafka connector 的 JAR 文件中，您无需手动安装。
+- 如果 StarRocks 表是主键表，则需要指定 addfield transform，用于向 Debezium CDC 格式数据的每个记录添加一个 `op` 字段。如果 StarRocks 表不是主键表，则无需指定 addfield transform。addfield transform 的类是 `com.Starrocks.Kafka.Transforms.AddOpFieldForDebeziumRecord`，已经包含在 Kafka connector 的 JAR 文件中，您无需手动安装。
```
Comment (yingtingdong) [inline docs/zh/loading/Kafka-connector-starrocks.md:240]: addfield transform 是把debezium格式的数据转成json或csv格式的数据，里面增加了__op字段用于标记上游数据对应的操作。这里好像写反了。

**Pull request merged**